src/evaluation.py

```diff
@@ -356,6 +360,23 @@ def conditionally_print_explanation(self):
         if self.should_explain_scoring:
             console.print(self.explanation_table, justify="center")
 
+    # Explanation Table to CSV
+    def conditionally_save_explanation_csv(self, evaluation_path):
+        if self.enable_evaluation_table_to_csv:
+            data = {col.header: col._cells for col in self.explanation_table.columns}
```

Comment:
Nice. Can you add a screenshot of what the output csv looks like now?

Reply:
![Screenshot from 2024-10-13 17-10-38](https://github.com/user-attachments/assets/a9958b65-115c-4d15-bc6d-b06b456e910c)
![Screenshot from 2024-10-13 17-11-53](https://github.com/user-attachments/assets/4ac24d29-8d72-4faf-9853-cc86b9b20bbf)

Reply:
@Drita-ai looks like the images are private/expired. Can you re-upload into the PR description?